homeassistant/components/nzbget/__init__.py

```diff
@@ -72,7 +72,9 @@ def setup(hass, config):
     scan_interval = config[DOMAIN][CONF_SCAN_INTERVAL]
 
     try:
-        nzbget_api = pynzbgetapi.NZBGetAPI(host, username, password, ssl, ssl, port)
+        nzbget_api = pynzbgetapi.NZBGetAPI(
+            host, username, password, ssl, ssl, port=port
```

Comment:
Signature of the method in question:

```
    def __init__(
        self,
        host,
        username=None,
        password=None,
        secure=False,
        verify_certificate=True,
        port=6789,
    ):
```

This PR isn't changing anything, as the port is the 6th parameter. Positional or by keyword would not make a difference.

Reply:
Would it be worth having all keyword parameters reference the keyword for safety? Seems strange to do it for one but not the rest